### ISSUE TITLE
Error on completing payment: Class 'HttpException' not found

### ISSUE THREAD

**Issue by j-greig** [enupal/paypal#2]
### Description
```
            if (!$order) {
                throw new \HttpException(400, Craft::t('enupal-paypal', 'Can not find the order for missing PayPal transaction id.'));
            }
``` 
----
```
Error: Class 'HttpException' not found in /Users/jamesgreig/ al/src/controllers/PaypalController.php:110 Stack trace: #0 [internal function]: enupal\paypal\controllers\PaypalController->actionCompletePayment() #1 /Users/jamesgreig/Repos/bernatklein/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array) #2 /Users/jamesgreig/Repos/bernatklein/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array) #3 /Users/jamesgreig/Repos/bernatklein/vendor/craftcms/cms/src/web/Controller.php(104): yii\base\Controller->runAction('complete-paymen...', Array) #4 /Users/jamesgreig/Repos/bernatklein/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('complete-paymen...', Array) #5 /Users/jamesgreig/Repos/bernatklein/vendor/craftcms/cms/src/web/Application.php(282): yii\base\Module->runAction('enupal-paypal/p...', Array) #6 /Users/jamesgreig/Repos/bernatklein/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('enupal-paypal/p...', Array) #7 /Users/jamesgreig/Repos/bernatklein/vendor/craftcms/cms/src/web/Application.php(271): yii\web\Application->handleRequest(Object(craft\web\Request)) #8 /Users/jamesgreig/Repos/bernatklein/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request)) #9 /Users/jamesgreig/Repos/bernatklein/web/index.php(21): yii\base\Application->run() #10 /Users/jamesgreig/.composer/vendor/laravel/valet/server.php(147): require('/Users/jamesgre...') #11 {main}
```

### Steps to reproduce

1. Complete a PayPal transaction
2. Page location changes back to Craft

### Additional info

- Craft version: 3.0.30.1
- PHP version: 7.1
- Database driver & version: MySQL
- Plugin version: 1.0.6

**Comment by andrelopez** [comment]
Hi @j-greig 

Thanks for reporting this bug.

Are you doing tests from a local env? you may need a public domain so the plugin can listen the IPN from PayPal. In this step the order is created once the payment is successfully.

We'll add a fix for the `HttpException` class in short.

**Comment by andrelopez** [comment]
Fixed in v1.0.7

**Comment by j-greig** [comment]
Thanks, it was indeed because I was testing locally :)
